README.md

```diff
@@ -242,6 +242,33 @@ const client = new faunadb.Client({
 })
 ```
 
+## Known issues
+
+### Using with Cloudflare Workers
+
+Cloudflare Workers has neither XMLHttpRequest nor fetch in global scope.
+Therefore, cross-fetch package is unable to inject fetch, and throws an error.
+fetch function is injected via closure, so the workaround would be to pass
+the fetch objects when initiating the FaunaDB client config. Cloudflare also
+doesn't support AbortController, which terminates requests as well as streams.
```

Comment:
```suggestion
doesn't support the use of an AbortController, which terminates requests as well as streams.
```
When you say "which terminates", the subject is ambiguous. Does an AbortController terminate requests/streams, or does the lack of an AbortController cause that?

Reply:
AbortController terminates requests. Cloudflare doesn't have AbortController. so even if a user decides to close a stream, it would be still alive

Reply:
Thanks for the information. I'm still not completely certain that I know how to write the last sentence to resolve the ambiguity; let me know if this sounds correct:

`... the use of an AbortController, which can cause request and stream termination without sufficient error handling.`